tests/metrics/statistical/test_wasserstein.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+
+import numpy as np
+
+import torch
+
+from scipy.stats import wasserstein_distance as sp_wasserstein
+from torcheval.metrics.statistical.wasserstein import Wasserstein1D
+from torcheval.utils.random_data import get_rand_data_wasserstein1d
+from torcheval.utils.test_utils.metric_class_tester import (
+    BATCH_SIZE,
+    MetricClassTester,
+    NUM_PROCESSES,
+)
+
+NUM_TOTAL_UPDATES = 8
+
+
+class TestWasserstein1D(MetricClassTester):
+    def _get_scipy_equivalent(
+        self,
+        x: torch.Tensor,
+        y: torch.Tensor,
+        x_weights: Optional[torch.Tensor] = None,
+        y_weights: Optional[torch.Tensor] = None,
+        device: str = "cpu",
+    ) -> torch.Tensor:
+        # Convert inputs to scipy style inputs
+        x_np = x.numpy().flatten()
+        y_np = y.numpy().flatten()
+        if x_weights is not None:
+            x_weights_np = x_weights.numpy().flatten()
+        if y_weights is not None:
+            y_weights_np = y_weights.numpy().flatten()
+
+        scipy_result = np.stack(
+            [
+                sp_wasserstein(sp_x, sp_y, sp_x_w, sp_y_w)
+                for sp_x, sp_y, sp_x_w, sp_y_w in zip(
+                    [x_np], [y_np], [x_weights_np], [y_weights_np]
+                )
```

Comment:
same as above, I think this is equivelant to 

```
[sp_wasserstein(x_np, y_np, x_weights_np, y_weights_np),]
```

Reply:
This change has been implemented, thanks!